### PR TITLE
surface: Return `Attached<WlSurface>` instead of WlSurface

### LINKED ISSUE #132: Why does create_surface/setup_surface do surface.detach?
This causes `thread 'main' panicked at 'Attemping to create an object from a non-attached proxy.'` when trying to get a frame callback from a sctk-created surface.

I guess I should've realized that it's not `Main` but I was completely baffled by this error, wondering what the hell is going on with my frame callback for quite some time >_<

### PR DESCRIPTION
**Pull request by kchibisov**
We should return `Attached<WlSurface>` from `create_surface`
and `create_surface_with_scale_callback` to simplify handling
of frame callbacks and object creation, since queue could be
unavailable in a place of surface creation.

Fixes #132.

### CODE REVIEW

File: src/surface.rs [hunk 1]
```diff
@@ -121,7 +121,7 @@ where
         }
     });
     surface.as_ref().user_data().set_threadsafe(|| Mutex::new(SurfaceUserData::new()));
-    surface.detach()
+    (*surface).clone()
```
Comment (kchibisov) [inline src/surface.rs:124]: In theory such thing could be implemented in wayland-client to convert `Main<I>` to `Attached<I>`, since `Main<I>` just encapsulates `Attached<I>`.

**Comment by kchibisov** [comment]
Fix for CI is #134

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 #### Changes
 
 - `andrew` is updated to `0.3`.
+- `create_surface` and `create_surface_with_scale_callback` now return `Attached<WlSurface>`
```
Comment (vberger) [inline CHANGELOG.md:8]: This is a breaking change, so it should be marked in a `#### Breaking Changes` section.
